test/JDBC/expected/getdate-vu-prepare.out

```diff
@@ -137,3 +137,128 @@ AS
     FROM Tally
     WHERE min_getdate = sys.getutcdate()
 GO
+
+CREATE PROCEDURE dbo.GetSysDatetimeDiff
+AS
+BEGIN
+    DECLARE @x datetime2 = SYSDATETIME();
+    select set_config('timezone', 'US/Pacific', false);
+    DECLARE @y datetime2 = SYSDATETIME()
+    select set_config('timezone', 'UTC', false);
+    SELECT DATEDIFF(MINUTE, @y, @x);
```

Comment:
should this be set time zone -> declare datetime2 -> set time zone --> declare datetime2 ?

Reply:
Or are we running the first line in default time zone ?

Reply:
Yes, first line is running in default time zone -> set time zone -> declare @y -> reset time zone -> Take DateDiff